package.json

```diff
@@ -38,6 +38,7 @@
     "postcss-loader": "~2.1.6",
     "prop-types": "~15.7.2",
     "react": "~16.14.0",
+    "react-bootstrap": "~1.5.2",
```

Comment:
We must [provide](https://react-bootstrap.github.io/getting-started/introduction#stylesheets) some stylesheet files for `react-bootstrap` to use internally, the opted solution here was to keep `bootstrap` as a dependency instead of adding a minified css file to the project or serve it via CDN.

For this reason, we must keep `jquery` and `popper` as dependencies too, as `bootstrap` needs them to be installed, although they're not being used for anything else.

Reply:
Is it possible to have react-bootstrap without jquery, somehow? Or a CSS only version of bootstrap as a dependency?

Reply:
Is jquery bundled in the frontend or it's just a npm dependency?

Reply:
Technically, `jquery` & `popper` aren't necessary, you can install and use the stylesheets from `bootstrap` on `react-bootstrap` without them. This works even during build.

The issue happens on CI when we run `npm ls` and it notices we haven't included them, so it crashes. One possible solution would be installing it as a separate command there (trying this now).

About a library that provides only the stylesheets, there's `bootstrap-sass` (used by other projects), but it only supports Bootstrap 3.